test/core/test/file-path.test.ts

```diff
@@ -0,0 +1,88 @@
+import { afterEach, describe, expect, it } from 'vitest'
+import { isWindows, slash, toFilePath } from '../../../packages/vite-node/src/utils'
+
+const originalCwd = process.cwd()
+
+function mockCwd(cwd: string) {
+  Object.defineProperty(process, 'cwd', {
```

Comment:
Can we use the mock from vitest directly?

Reply:
let me try, I wrote like this is because I remember document mentioned that the mocking cannot work on Node.js built-in API

Reply:
turned out that I should try the built-in mock first next time 🤣